Automattic/woocommerce-payments
# Collect border styles from the primary container element

Fixes https://github.com/Automattic/woocommerce-payments/issues/9507

#### Changes proposed in this Pull Request

Pass styling data from the element that contains container borders to WooPay.








#### Testing instructions
1. Go to your WooPay site /wp-admin > WC Pay dev.
2. Check the Enable WooPay global theme support option.
3. In your merchant site make sure [this function](https://github.com/Automattic/woocommerce-payments/blob/5c2c8a1b4afa5c64246ca7f07194ea9d59688dd4/includes/class-wc-payment-gateway-wcpay.php#L2573) return true or you have enabled WooPay theming in your local WCPay server.
4. Go through WooPay checkout and make sure the border styles of the WooPay checkout match merchant checkout.
5. For detailed testing instructions follow https://github.com/Automattic/woopay/pull/2942






-------------------


**Post merge**
